src/client_table.cpp

```diff
@@ -0,0 +1,191 @@
+#include <cstdint>
```

Comment:
Add license header

Reply:
Also, maybe move this file to a example directory?

Reply:
yes